### ISSUE TITLE
undefined return values ?

### ISSUE THREAD

**Issue by netvisao** [Netflix/dynomite#216]
```
dyn_client.c:289:13: error: non-void function 'client_handle_response' should
      return a value [-Wreturn-type]
            return;
            ^
dyn_client.c:295:9: error: non-void function 'client_handle_response' should
      return a value [-Wreturn-type]
        return;
        ^
```

**Comment by shailesh33** [comment]
Will take a look. Some day I will work on all the compilation warnings that come up.

**Comment by shailesh33** [comment]
Fixed in release/v0.5.6 branch. Will be in master soon.
